src/core/server/saved_objects/service/lib/search_dsl/search_dsl.ts

```diff
@@ -73,5 +78,7 @@ export function getSearchDsl(
       kueryNode,
     }),
     ...getSortingParams(mappings, type, sortField, sortOrder),
+    ...getPitParams(pit),
```

Comment:
NIT: I would go with `...(pit ? getPitParams(pit) : {})` instead of having the `pit` param of `getPitParams` being optional